tests/gutenberg_ro.e2e.spec.js

```diff
@@ -0,0 +1,360 @@
+                                console.log(supported ? '' : '\x1b[33m%s\x1b[0m', '      ' + alertText);
+                                return supported;
+                            })
+                        }
+                    }
+                    const approveButton = await driver.findElement(By.id('approveConfirm'));
+                    await approveButton.click();
+                } catch (e) {
+                    // Do nothing
+                }
+                if (mode === 'jquery' || serviceWorkerAPI) {
+                    // Wait until the mode has switched
+                    await driver.sleep(500);
+                    let serviceWorkerStatus = await driver.findElement(By.id('serviceWorkerStatus')).getText();
+                    try {
+                        if (mode === 'serviceworker') {
+                            assert.equal(true, /and\sregistered/i.test(serviceWorkerStatus));
+                        } else {
+                            assert.equal(true, /not\sregistered|unavailable/i.test(serviceWorkerStatus));
+                        }
+                    } catch (e) {
+                        if (!~modes.indexOf('serviceworker')) {
+                            // We can't switch to serviceworker mode if it is not being tested, so we should fail the test
+                            throw e;
+                        }
+                        // We failed to switch modes, so let's try switching back and switching to this mode again
+                        console.log('\x1b[33m%s\x1b[0m', '      Failed to switch to ' + mode + ' mode, trying again...');
+                        const otherModeSelector = await driver.findElement(By.id(mode === 'jquery' ? 'serviceworkerModeRadio' : 'jqueryModeRadio'));
+                        // Click the other mode selector
+                        await otherModeSelector.click();
+                        // Wait until the mode has switched
+                        await driver.sleep(330);
+                        // Click the mode selector again
+                        await modeSelector.click();
+                        // Wait until the mode has switched
+                        await driver.sleep(330);
+                        serviceWorkerStatus = await driver.findElement(By.id('serviceWorkerStatus')).getText();
+                        if (mode === 'serviceworker') {
+                            assert.equal(true, /and\sregistered/i.test(serviceWorkerStatus));
+                        } else {
+                            assert.equal(true, /not\sregistered|unavailable/i.test(serviceWorkerStatus));
+                        }
+                    }
+                } else {
+                    // Skip remaining SW mode tests if the browser does not support the SW API
+                    console.log('\x1b[33m%s\x1b[0m', '      Skipping SW mode tests...');
+                    await driver.quit();
```

Comment:
Make sure the Promise is returned when we exit, with `return await driver.quit();`

Reply:
This needs to be `return await driver.quit();` or else maybe `return driver.quit()` is enough (if it's alreadya a Promise).

Reply:
`return await driver.quit();` will actually return a `void` not `Promise`